scripts/effects/mounted.lua

```diff
@@ -23,6 +23,11 @@ end
 
 effectObject.onEffectLose = function(target, effect)
     target:setAnimation(xi.anim.NONE)
+
+    -- Remove CharVars from player participating in chocobo riding game
+    if target:isPC() then
```

Comment:
Could mounted ever apply to a non-PC?

Reply:
0x00E supports it, technically. There are mounted goblins in events, too. tldr maybe

Reply:
Happy to remove it if needed, just was playing it safe in case there was a situation I wasn't familiar with in retail.